src/test/java/seedu/address/commons/util/StringUtilTest.java

```diff
@@ -50,77 +50,77 @@ public void isNonZeroUnsignedInteger() {
 
     /*
      * Invalid equivalence partitions for word: null, empty, multiple words
-     * Invalid equivalence partitions for sentence: null
+     * Invalid equivalence partitions for ModuleCode: null
      * The four test cases below test one invalid input at a time.
      */
 
     @Test
     public void containsWordIgnoreCase_nullWord_throwsNullPointerException() {
-        assertThrows(NullPointerException.class, () -> StringUtil.containsWordIgnoreCase("typical sentence", null));
+        assertThrows(NullPointerException.class, () -> StringUtil.containsCharSequenceIgnoreCase(
+                "typical ModuleCode", null));
     }
 
     @Test
     public void containsWordIgnoreCase_emptyWord_throwsIllegalArgumentException() {
-        assertThrows(IllegalArgumentException.class, "Word parameter cannot be empty", ()
-            -> StringUtil.containsWordIgnoreCase("typical sentence", "  "));
+        assertThrows(IllegalArgumentException.class, "CharSequence parameter cannot be empty", () ->
+                StringUtil.containsCharSequenceIgnoreCase("typical ModuleCode", "  "));
     }
 
     @Test
     public void containsWordIgnoreCase_multipleWords_throwsIllegalArgumentException() {
-        assertThrows(IllegalArgumentException.class, "Word parameter should be a single word", ()
-            -> StringUtil.containsWordIgnoreCase("typical sentence", "aaa BBB"));
+        assertThrows(IllegalArgumentException.class, "CharSequence parameter should be a single word", () ->
+                StringUtil.containsCharSequenceIgnoreCase("typical ModuleCode", "aaa BBB"));
     }
 
     @Test
-    public void containsWordIgnoreCase_nullSentence_throwsNullPointerException() {
-        assertThrows(NullPointerException.class, () -> StringUtil.containsWordIgnoreCase(null, "abc"));
+    public void containsWordIgnoreCase_nullModuleCode_throwsNullPointerException() {
+        assertThrows(NullPointerException.class, () -> StringUtil.containsCharSequenceIgnoreCase(null, "abc"));
     }
 
     /*
-     * Valid equivalence partitions for word:
+     * Valid equivalence partitions for CharSequence:
      *   - any word
      *   - word containing symbols/numbers
      *   - word with leading/trailing spaces
      *
-     * Valid equivalence partitions for sentence:
+     * Valid equivalence partitions for ModuleCode:
      *   - empty string
      *   - one word
      *   - multiple words
```

Comment:
Do you think we should remove this ah? I feel like it might be an irrelevant equivalence class now that we use a `.contains()` check

Removing it will also match the new test cases below since we're removing the old tests involving multiple words

Reply:
yup agree!